samples/app-checkin-location/nodejs/index.js

```diff
@@ -34,6 +34,7 @@ adapter.onTurnError = async (context, error) => {
     //       application insights. See https://aka.ms/bottelemetry for telemetry 
     //       configuration instructions.
     console.error(`\n [onTurnError] unhandled error: ${error}`);
+    await context.sendTraceActivity(`Sorry, it looks like something went wrong. Exception Caught: ${error}`);
```

Comment:
This is not needed, Why we are using sendTraceActivity? because below line same is available.

Reply:
after below line need to add that code in commented mode.
 await context.sendActivity('To continue to run this bot, please fix the bot source code.');

Reply:
resolved